src/databricks/labs/ucx/source_code/base.py

```diff
@@ -136,6 +137,35 @@ class Linter:
     def lint(self, code: str) -> Iterable[Advice]: ...
 
 
+class SQLLinter(Linter):
```

Comment:
```suggestion
class QueryLinter(Linter):
```
is this easier to read? `LL` reads/sounds difficult

Reply:
How about SqlLinter ? (`CREATE TABLE xxx` is not a query)

Reply:
done